src/esp/assets/ResourceManager.h

```diff
@@ -338,6 +340,32 @@ class ResourceManager {
                                scene::SceneNode& node,
                                DrawableGroup* drawables);
 
+  /**
+   * @brief Set a named @ref LightSetup
+   *
+   * If this name already exists, the @ref LightSetup is updated and all @ref
+   * Drawables using this setup are updated.
+   *
+   * @param key Key to identify this @ref LightSetup
+   * @param setup Light setup this key will now references
+   */
+  void setLightSetup(const Magnum::ResourceKey& key, gfx::LightSetup setup);
+
+  /**
+   * @brief Get a named @ref LightSetup
+   */
+  Magnum::Resource<gfx::LightSetup> getLightSetup(
+      const Magnum::ResourceKey& key);
+
+  //! @brief The @ref ShaderManager key for @ref LightInfo which has no lights
+  static constexpr char NO_LIGHT_KEY[] = "no_lights";
+
+  //! @brief The @ref ShaderManager key for the default @ref LightInfo
+  static constexpr char DEFAULT_LIGHTING_KEY[] = "default_lighting";
+
+  //! @brief The @ref ShaderManager key for the default @ref MaterialInfo
+  static constexpr char DEFAULT_MATERIAL_KEY[] = "default_material";
```

Comment:
I want these to be exported so the defaults can be updated without hardcoding the "default" string everywhere that does this. eg. `Simulator`, and `Viewer` do this. For the `SHADER_KEY`, they don't really need to be in the headers, but personally I like that since it makes a collision down the line easier to debug without looking through the implementation files.

Reply:
I think it's fine to leave these explicit for now.